### ISSUE TITLE
Favicon get path

### ISSUE THREAD

**Issue by jossmac** [keystonejs/keystone#2743]
CreateApp appears to have a typo when retrieving the `favicon` path - [code snippet](https://github.com/keystonejs/keystone/blob/0e5a4a11d13d4ced058b392b119140f42116dc7e/server/createApp.js#L43-L45)

```JS
if (keystone.get('favico')) {
  app.use(favicon(keystone.getPath('favico')));
}
```

Should be looking for `favicon` **NOT** `favico`. Also see [Keystone docs reference](http://keystonejs.com/docs/configuration/#options-server)

@mxstbr please confirm or close and advise

**Comment by mxstbr** [comment]
Definitely a bug, would you mind submitting a PR?

**Comment by jossmac** [comment]
Fixed by 8921cf7
